modules/nw-metallb-operator-initial-config.adoc

```diff
@@ -0,0 +1,59 @@
+[id="nw-metallb-operator-initial-config_{context}"]
+= Starting MetalLB on your cluster
+
+After you install the Operator, you need to configure a single instance of a MetalLB custom resource. After you configure the custom resource, the Operator starts MetalLB on your cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
```

Comment:
'MetalLB operator already installed' could be listed as item in prerequisite even though section introduction states it.